### DISCUSSION TITLE
Need help with Food Delivery Schema

### DISCUSSION THREAD

**Discussion by frsmlk** [prisma/prisma#17648] [Q&A] [answered]
Does this prisma model make sense?

Here's what i want. There are multiple dishes related to MenuOnDay and there are multiple MenuOnDay related to MenuOnSchedule.

I'm not sure what MenuOnDay[] in Dishes and Dishes[] in MenuOnDay means.

In simple english, MenuOnSchedule is a weekly schedule that is made out of 5 daily menus (Monday to Friday) which is the MenuOnDay model. MenuOnDay is made out of all the Dishes that will be served that day. 

Would appreciate your advice on best practice for this.  

```
model Dishes {
  id          String      @id @default(uuid())
  createdAt   DateTime    @default(now())
  updatedAt   DateTime    @updatedAt
  name        String      @unique
  category    String
  subCategory String
  image       String
  cuisine     String
  description String
  MenuOnDay   MenuOnDay[]
}

model MenuOnDay {
  id               String          @id @default(uuid())
  createdAt        DateTime        @default(now())
  updatedAt        DateTime        @updatedAt
  date             DateTime
  Dishes           Dishes[]
  MenuOnSchedule   MenuOnSchedule? @relation(fields: [menuOnScheduleId], references: [id])
  menuOnScheduleId String?
}

model MenuOnSchedule {
  id        String      @id @default(uuid())
  createdAt DateTime    @default(now())
  updatedAt DateTime    @updatedAt
  date      DateTime
  startDate DateTime
  endDate   DateTime
  menu      MenuOnDay[]
}
```

**Comment by nurul3101** [comment, marked as answer]
Hey @frsmlk 👋 

> I'm not sure what MenuOnDay[] in Dishes and Dishes[] in MenuOnDay means.

MenuOnDay[] in Dishes would mean that a single Dish can be a part of multiple MenuOnDay's.
Dishes[] in MenuOnDay would mean that a single MenuOnDay can have multiple Dishes.

Please note that as these fields are relation fields they exist only at Prisma level and not at Database level. So there won't be any columns like Dishes in the MenuOnDay table and MenuOnDay column in the Dishes table.

Apart from that the models look good to me.

**Reply by frsmlk** [reply]
Thanks for the quick reply. 

I'd still be able to make this query right?

Find MenuOnSchedule for this week and populate MenuOnday and Dishes

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
